src/sentry/api/endpoints/user_emails.py

```diff
@@ -60,10 +60,11 @@ def add_email(email, user, subscribe_newsletter=False):
 
         # Update newsletter subscription and mark as unverified
         if subscribe_newsletter:
-            newsletter.update_subscription(user=user,
-                                           verified=False,
-                                           list_id=1,
-                                           )
+            newsletter.update_subscription(
```

Comment:
should this be create_or_update?

Reply:
`newsletter.create_or_update_subscription(...)` ya